app/models/user.rb

```diff
@@ -265,7 +265,8 @@ def missing_require_fields?
   def send_welcome_email
     return if role?('guest') || role?('preview_session')
 
-    UserMailer.welcome_email(human_readable_role, email).deliver_later
+    mailer = role?('participant') && interventions.present? ? UserMailer.with(locale: interventions.first.language_code) : UserMailer
```

Comment:
Participant doesn't have a relationship with interventions, it is only for researcher